OpenSim/Examples/ReadAPDMData/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_executable(readAPDMData readAPDMData.cpp)
+target_link_libraries(readAPDMData ${Simbody_LIBRARIES} osimCommon)
+set_target_properties(readAPDMData
+       PROPERTIES        
+       PROJECT_LABEL "Utilities - readAPDMData"
```

Comment:
Is this a utility or an example?

Reply:
It should be a moot point once we refactor, it started as a Utility but there's no place for common utilities so I just piggybacked on whatever examples are there for the short term.